### DISCUSSION TITLE
receive emails sent to postmaster or abuse

### DISCUSSION THREAD

**Discussion by csatsias** [postalserver/postal#2346] [Help with using Postal] [answered]
We are trying to register to feedback loops (e.g. validity https://fbl.validity.com/manage) and they are asking us to choose emails between postmaster@ourdomain.com or abuse@ourdomain.com . We cannot create such email addresses since both of them are reserved words. Could you please advice us how we could do this on postal so that we can receive the emails sent to these two addresses?

**Comment by willpower232** [comment, marked as answer]
Postal could only help you if you directed all email for ourdomain.com to it and I would presume you don't want that.

If you do want Postal to handle your incoming mail, you can set up routes to forward messages it receives to other mailboxes or services over HTTP

![image](https://user-images.githubusercontent.com/1619102/216008365-9b946fe6-673e-4339-87cb-df876ca310cc.png)
